src/head/script.js

```diff
@@ -1,95 +1,178 @@
+			(function () {
+				const gaALt = document.createElement('script'); gaALt.type = 'text/javascript'; gaALt.async = true;
+				gaALt.src = `${document.location.protocol == 'https:' ? 'https://ssl' : 'http://www'}.google-analytics.com/ga.js`;
+				const s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(gaALt, s);
+			}());
 		}
+
+		(function (i, s, o, g, r, a, m) {
+			i.GoogleAnalyticsObject = r; i[r] = i[r] || function () {
+				(i[r].q = i[r].q || []).push(arguments);
+			}, i[r].l = 1 * new Date(); a = s.createElement(o),
+			m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m);
+		}(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga'));
+		ga('create', config.gaId, 'auto');
+		/* eslint-enable */
+	};
+
+	// One Trust Cookie Management
+	if (config.oneTrust && config.oneTrust.enable) {
+		/* eslint-disable */
+		(function () {
+			// Main OneTrust script to display cookie notice and set user preferences
+			const p = document.getElementsByTagName('script')[0];
+			const s = document.createElement('script');
+			s.setAttribute('type', 'text/javascript');
+			s.setAttribute('data-domain-script', `${config.oneTrust.key}${config.oneTrust.domainSuffix}`)
+			s.src = `https://cdn.cookielaw.org/consent/${config.oneTrust.key}${config.oneTrust.domainSuffix}/otSDKStub.js`;
+			p.parentNode.insertBefore(s, p);
+		}());
+
+		// Follow up OneTrust function.
+		// This function is called when cookie preferences are changed on the OneTrust widget
+		window.OptanonWrapper = () => {
+			/**
+			 * Insert scripts if user has enabled OneTrust Category
+			 * These categories must match the categories for the specific cookies in the OneTrust admin interface
+			 * */
+
+			/** Category 'C0001'
+			* Strictly Necessary Cookies
+			* These cookies are necessary for the website to function and cannot be switched off in our systems.
+			* They are usually only set in response to actions made by you which amount to a request for services,
+			* such as setting your privacy preferences, logging in or filling in forms. You can set your browser
+			* to block or alert you about these cookies, but some parts of the site will not then work.
+			* These cookies do not store any personally identifiable information.
+			* */
+
+			/** Category 'C0002'
+			* Performance Cookies
+			* These cookies allow us to count visits and traffic sources so we can measure and improve the performance
+			* of our site. They help us to know which pages are the most and least popular and see how visitors move
+			* around the site. All information these cookies collect is aggregated and therefore anonymous. If you
+			* do not allow these cookies we will not know when you have visited our site, and will not be able to
+			* monitor its performance.
+			* */
+			if (config.enableAnalytics && config.enableGA) {
+				OneTrust.InsertHtml('', 'head', insertGoogleAnalytics, null, 'C0002');
```

Comment:
This is kind of a hack because the OneTrust API is pretty limited. You only get InsertScript, which doesn't work unless the script is in an external file, and would require some refactoring of the way we load the external libraries. The InsertHtml method here, inserts an empty string into the head of the document, and then calls our callback function